### ISSUE TITLE
Import broken by sqlalchemy 1.4.0

### ISSUE THREAD

**Issue by NakulK48** [kvesteri/sqlalchemy-utils#505]
```
[2021-03-15T18:09:18.021Z] ../../../miniconda/envs/jenkins-jca_multibranch-PR-52-1/lib/python3.8/site-packages/sqlalchemy_utils/functions/orm.py:14: in <module>

[2021-03-15T18:09:18.021Z]     from sqlalchemy.orm.query import _ColumnEntity

[2021-03-15T18:09:18.021Z] E   ImportError: cannot import name '_ColumnEntity' from 'sqlalchemy.orm.query' (/home/jcauser/miniconda/envs/jenkins-jca_multibranch-PR-52-1/lib/python3.8/site-packages/sqlalchemy/orm/query.py)
```

**Comment by bahamut45** [comment]
👍 same error

**Comment by aduggirala28** [comment]
Same issue, error with importing  '_ColumnEntity'
```
Name: SQLAlchemy
Version: 1.4.0
Summary: Database Abstraction Library
Home-page: http://www.sqlalchemy.org
Author: Mike Bayer
Author-email: mike_mp@zzzcomputing.com
License: MIT
Requires: greenlet
Required-by: SQLAlchemy-Utils, Flask-SQLAlchemy, alembic
(my_oar) aduggirala@C02ZM1PPLVDM ouroath.oar % 
``` 

```
Name: SQLAlchemy-Utils
Version: 0.36.8
Summary: Various utility functions for SQLAlchemy.
Home-page: https://github.com/kvesteri/sqlalchemy-utils
Author: Konsta Vesterinen, Ryan Leckey, Janne Vanhala, Vesa Uimonen
Author-email: konsta@fastmonkeys.com
License: BSD
Requires: six, SQLAlchemy
Required-by: 
```

**Comment by jbvsmo** [comment]
Can you guys make a quick fix by adding requirement version < 1.4.0? Otherwise if people start doing it they might get stuck with old sqlalchemy

**Comment by nsoranzo** [comment]
Duplicate of https://github.com/kvesteri/sqlalchemy-utils/issues/474

**Comment by NakulK48** [comment]
> Duplicate of #474

`1.4` left beta yesterday, so I think this is more urgent now than it was.

**Comment by sphuber** [comment]
There is another problem:
```
/opt/hostedtoolcache/Python/3.8.8/x64/lib/python3.8/site-packages/sqlalchemy_utils/expressions.py:4: in <module>
    from sqlalchemy.sql.expression import (
E   ImportError: cannot import name '_literal_as_text' from 'sqlalchemy.sql.expression' (/opt/hostedtoolcache/Python/3.8.8/x64/lib/python3.8/site-packages/sqlalchemy/sql/expression.py)
```
This is because of [this commit in `sqlalchemy`](https://github.com/sqlalchemy/sqlalchemy/commit/f07e050c9ce4afdeb9c0c136dbcc547f7e5ac7b8).

**Comment by waghmareomkar** [comment]
> ```
> [2021-03-15T18:09:18.021Z] ../../../miniconda/envs/jenkins-jca_multibranch-PR-52-1/lib/python3.8/site-packages/sqlalchemy_utils/functions/orm.py:14: in <module>
> 
> [2021-03-15T18:09:18.021Z]     from sqlalchemy.orm.query import _ColumnEntity
> 
> [2021-03-15T18:09:18.021Z] E   ImportError: cannot import name '_ColumnEntity' from 'sqlalchemy.orm.query' (/home/jcauser/miniconda/envs/jenkins-jca_multibranch-PR-52-1/lib/python3.8/site-packages/sqlalchemy/orm/query.py)
> ```

it worked for me when i installed using conda instead of pip. Hope it works for you aswell.

**Comment by mgrandi** [comment]
you can temporarily fix this by changing the imports in `sqlalchemy_utils\functions\orm.py`:

`from sqlalchemy.orm.query import _ColumnEntity`

to

`from sqlalchemy.orm.context import _ColumnEntity`

**Comment by namelivia** [comment]
It seems that #506 is being in the works to fix this

**Comment by WolfgangFahl** [comment]
see https://github.com/sqlalchemy/sqlalchemy/issues/6226 and https://stackoverflow.com/a/66648061/1497139
